app/services/scenes/scene-folder.ts

```diff
@@ -26,16 +27,20 @@ export class SceneItemFolder extends SceneItemNode {
 
     const state = this.scenesService.state.scenes[sceneId].nodes.find(item => {
       return item.id === id;
-    });
+    })!;
```

Comment:
I think it's incorrect to assert not-null here.  It is entirely possible that I could do `new SceneItemFolder(sceneId, badNodeId)`.  We should handle with a runtime exception.